About desktop

grytsayo opened the pull request:
Desktop layout done

grytsayo committed:
About desktop

Desktop layout done

grytsayo committed:
Bugfix tablet version

.about-section
display: flex
align-items: center

.content__item
line-height commented

grytsayo committed:
Bugfix picture

Optimized picture download done

grytsayo committed:
Btn read more/less done

Btn read more/less done

grytsayo commented:
It seems work more or less)

ohrimovich commented on src/partials/about-us.html:
```diff
@@ -27,12 +27,39 @@ <h2 class="content__title"  >
+            moreText.style.display = "none";
+          } else {
+            dots.style.display = "none";
+            btnText.innerHTML = "Read less"; 
```
По шаблону кнопка less

ohrimovich commented on src/sass/_about-us.scss:
```diff
@@ -1,20 +1,18 @@
-        max-width: 329px;   
-    }
-
+    max-width: 100%;
```
Ці стилі зайві
вони задані в _base.scss для тегу img

grytsayo committed:
Bugfix

==html
JS script commented
List to dev changed
==CSS
Width atr deleted from picture
LastChild display none for desktop
